### ISSUE TITLE
Prompt for targetCluster and targetNamespace

### ISSUE THREAD

**Issue by fenech** [Azure/vscode-bridge-to-kubernetes#86]
**Is your feature request related to a problem? Please describe.**

After the initial configuration of the extension, a task is created with targetCluster and targetNamespace set. This means that, to target a different cluster/namespace, either the extension should be reconfigured, or the tasks.json file edited manually.

**Mention what platform you want to support the new feature**

vscode

**Describe the solution you'd like**

I'd like to be prompted to choose the targetCluster and targetNamespace, if they haven't been set to a fixed value.

**Describe alternatives you've considered**

```json
{
  "version": "2.0.0",
  "tasks": [
    {
      "label": "bridge-to-kubernetes.resource",
      "type": "bridge-to-kubernetes.resource",
      "resource": "nearbyone-device-manager",
      "resourceType": "service",
      "ports": [5300],
      "targetCluster": "${input:targetCluster}",
      "targetNamespace": "${input:targetNamespace}",
      "useKubernetesServiceEnvironmentVariables": false,
      "targetContainer": "device-manager"
    }
  ],
  "inputs": [
    {
      "id": "targetCluster",
      "description": "Target Kubernetes cluster",
      "type": "promptString"
    },
    {
      "id": "targetNamespace",
      "description": "Target Namespace",
      "type": "promptString"
    }
  ]
}
```

I've tried using `${input:}` placeholders, and was prompted as expected, but then I got an error message, since it appears that:
* the placeholders weren't expanded
* the targetCluster and targetNamespace need to match the current context, as set in the Kubernetes extension

I also tried to make this task depend on another task to select the current k8s context, but was unsuccessful in doing so.

**Additional context**

![screenshot showing unexpanded placeholders in terminal](https://github.com/Azure/vscode-bridge-to-kubernetes/assets/3943705/53b33ab6-a412-49c4-bd7b-9a0b994686a5)
![screenshot showing unexpanded placeholders in dialog box](https://github.com/Azure/vscode-bridge-to-kubernetes/assets/3943705/83be86af-951a-4d2c-8b41-306fb095f049)

To be honest, I'm not 100% clear on the role of these parameters, if indeed they should always match the settings in the Kubernetes extension.

**Comment by hsubramanianaks** [comment]
@fenech Thanks for this request, initially I thought it is an enhancement and looked into this but seems like not an enhancement. bridge to kubernetes is dependant on k8s extension (left panel) in vscode and cluster, namespace selection is also depending on k8s extension. if you have to switch you have to use k8s extension to choose a different cluster and namespace. Because b2k relies on the current kube context's which can be modified via the k8s extension. Here is the code reference [here](https://github.com/Azure/vscode-bridge-to-kubernetes/blob/main/src/connect/ConnectWizard.ts#L221).

The parameters targetNamespace and targetCluster are required by the b2k extension to pass it to the b2k CLI which actually starts the debug/connect session for the user. And these parameters are received from k8s extension. 

![image](https://github.com/Azure/vscode-bridge-to-kubernetes/assets/105889062/e4b0a4a8-c964-4dcb-8e47-abad9b8abcef)


If you still think this feature needs to be present in b2k extension, please let me know reason or purpose for this so that I can take this back to my team to see if this can be done. Thank you.
